node/full_client.go

```diff
@@ -25,6 +25,7 @@ import (
 	rconfig "github.com/celestiaorg/rollmint/config"
 	abciconv "github.com/celestiaorg/rollmint/conv/abci"
 	"github.com/celestiaorg/rollmint/mempool"
+	rtypes "github.com/celestiaorg/rollmint/types"
```

Comment:
I would prefer to use just `types` here, and alias `"github.com/tendermint/tendermint/types"` as `tmtypes`.